index.js

```diff
@@ -14,27 +14,37 @@ token_utils._tokenCache = tokenCache; // Exposed for testing
 // This key can them be used to verify the JWT token presented so that the
 // details contained within the token can be trusted.  Such as the user, expiry and scopes.
 const getKey = (keyURL) => {
```

Comment:
Looks like you missed the added parameter

Reply:
sorry.. copy paste mistake between branches. fixed